mdata/cache/accnt/if.go

```diff
@@ -0,0 +1,24 @@
+package cache
+
+type Accnt interface {
+	Add(string, uint32, uint64)
+	Hit(string, uint32)
+	GetEvictQ() chan *EvictTarget
+}
+
+type EvictTarget struct {
+	Metric string
+	Ts     uint32
+}
+
+func NewAccnt(maxSize uint64) Accnt {
```

Comment:
ok, did that. there's no `FlatAccnt.init()` anymore, it's all in `NewFlatAccnt`.